### DISCUSSION TITLE
Support new Quarkus 3.3.0

### DISCUSSION THREAD

**Discussion by miguelborges99** [quarkiverse/quarkiverse#171] [Q&A]
Hi @gsmet / @gastaldi,

Quarkus 3.3.0 CR1 was released some weeks ago, and it come with some changes that broke the [quarkus-reactive-h2-client](https://github.com/quarkiverse/quarkus-reactive-h2-client). I've made a [PR](https://github.com/quarkiverse/quarkus-reactive-h2-client/pull/30) to solve it. Hope you find it useful.

**Comment by gastaldi** [comment]
Looks like this was already merged. Closing
